tools/monorepo-utils/src/slack/commands/slack/slack-message.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { Command } from '@commander-js/extra-typings';
+
+/**
+ * Internal dependencies
+ */
+import { Logger } from '../../../core/logger';
+import { requestAsync } from '../../../core/util';
+
+type SlackResponse = {
+	ok: boolean;
+	error?: string;
+};
+
+export const slackMessageCommand = new Command( 'message' )
+	.description( 'Send a plain-text message to a slack channel' )
+	.argument(
+		'<token>',
+		'Slack authentication token bearing required scopes.'
+	)
+	.argument( '<channel>', 'Slack channel to send the message to.' )
+	.argument( '<text>', 'Text based message to send to the slack channel.' )
```

Comment:
We could add other options from the slack API later (such as icon and others), but for now we only need this basic functionality.